ramp-tps/src/utils.rs

```diff
@@ -0,0 +1,77 @@
+use bzip2::bufread::BzDecoder;
+use solana_sdk::genesis_block::GenesisBlock;
+use solana_sdk::timing::duration_as_ms;
+use std::fs::File;
+use std::io;
+use std::net::SocketAddr;
+use std::path::Path;
+use std::thread::sleep;
+use std::time::Duration;
+use std::time::Instant;
+use tar::Archive;
+
+const GENESIS_ARCHIVE_NAME: &str = "genesis.tar.bz2";
+
+/// Inspired by solana_local_cluster::cluster_tests
+pub fn sleep_n_slots(num_slots: u64, genesis_block: &GenesisBlock) {
+    let poh_config = &genesis_block.poh_config;
+    let ticks_per_slot = genesis_block.ticks_per_slot;
+    let num_ticks_to_sleep = num_slots as f64 * ticks_per_slot as f64;
+    let num_ticks_per_second = (1000 / duration_as_ms(&poh_config.target_tick_duration)) as f64;
+    let secs = ((num_ticks_to_sleep + num_ticks_per_second - 1.0) / num_ticks_per_second) as u64;
```

Comment:
What's the reasoning behind this line? (ie. why the extra most-of-a-second?)

Reply:
Looks to me like it guarantees that secs doesn't get rounded down (this code was borrowed from elsewhere)